traffic_ops/testing/api/v2/deliveryservices_test.go

```diff
@@ -0,0 +1,602 @@
+		t.Errorf("cannot GET DeliveryServices: %v - %v", err, dses)
+	}
+	ds := tc.DeliveryServiceNullable{}
+	for _, d := range dses {
+		if *d.XMLID == testDS.XMLID {
+			ds = d
+			break
+		}
+	}
+	// GET latest, verify expected values for 1.3 and 1.4 fields
+	if ds.DeepCachingType == nil {
+		t.Errorf("expected DeepCachingType: %s, actual: nil", testDS.DeepCachingType.String())
+	} else if *ds.DeepCachingType != testDS.DeepCachingType {
+		t.Errorf("expected DeepCachingType: %s, actual: %s", testDS.DeepCachingType.String(), ds.DeepCachingType.String())
+	}
+	if ds.FQPacingRate == nil {
+		t.Errorf("expected FQPacingRate: %d, actual: nil", testDS.FQPacingRate)
+	} else if *ds.FQPacingRate != testDS.FQPacingRate {
+		t.Errorf("expected FQPacingRate: %d, actual: %d", testDS.FQPacingRate, *ds.FQPacingRate)
+	}
+	if ds.SigningAlgorithm == nil {
+		t.Errorf("expected SigningAlgorithm: %s, actual: nil", testDS.SigningAlgorithm)
+	} else if *ds.SigningAlgorithm != testDS.SigningAlgorithm {
+		t.Errorf("expected SigningAlgorithm: %s, actual: %s", testDS.SigningAlgorithm, *ds.SigningAlgorithm)
+	}
+	if ds.Tenant == nil {
+		t.Errorf("expected Tenant: %s, actual: nil", testDS.Tenant)
+	} else if *ds.Tenant != testDS.Tenant {
+		t.Errorf("expected Tenant: %s, actual: %s", testDS.Tenant, *ds.Tenant)
+	}
+	if ds.TRRequestHeaders == nil {
+		t.Errorf("expected TRRequestHeaders: %s, actual: nil", testDS.TRRequestHeaders)
+	} else if *ds.TRRequestHeaders != testDS.TRRequestHeaders {
+		t.Errorf("expected TRRequestHeaders: %s, actual: %s", testDS.TRRequestHeaders, *ds.TRRequestHeaders)
+	}
+	if ds.TRResponseHeaders == nil {
+		t.Errorf("expected TRResponseHeaders: %s, actual: nil", testDS.TRResponseHeaders)
+	} else if *ds.TRResponseHeaders != testDS.TRResponseHeaders {
+		t.Errorf("expected TRResponseHeaders: %s, actual: %s", testDS.TRResponseHeaders, *ds.TRResponseHeaders)
+	}
+	if ds.ConsistentHashRegex == nil {
+		t.Errorf("expected ConsistentHashRegex: %s, actual: nil", testDS.ConsistentHashRegex)
+	} else if *ds.ConsistentHashRegex != testDS.ConsistentHashRegex {
+		t.Errorf("expected ConsistentHashRegex: %s, actual: %s", testDS.ConsistentHashRegex, *ds.ConsistentHashRegex)
+	}
+	if ds.ConsistentHashQueryParams == nil {
+		t.Errorf("expected ConsistentHashQueryParams: %v, actual: nil", testDS.ConsistentHashQueryParams)
+	} else if !reflect.DeepEqual(ds.ConsistentHashQueryParams, testDS.ConsistentHashQueryParams) {
+		t.Errorf("expected ConsistentHashQueryParams: %v, actual: %v", testDS.ConsistentHashQueryParams, ds.ConsistentHashQueryParams)
+	}
+	if ds.MaxOriginConnections == nil {
+		t.Errorf("expected MaxOriginConnections: %d, actual: nil", testDS.MaxOriginConnections)
+	} else if *ds.MaxOriginConnections != testDS.MaxOriginConnections {
+		t.Errorf("expected MaxOriginConnections: %d, actual: %d", testDS.MaxOriginConnections, *ds.MaxOriginConnections)
+	}
+
+	// GET 1.1, verify 1.3 and 1.4 fields are nil
+	data := tc.DeliveryServicesNullableResponse{}
```

Comment:
This whole bit about making a request to the v1.1 API and checking that the newer fields are nil shouldn't be included, because this can't/won't test the v1.1 API